streams/jo.m3u

```diff
@@ -1,6 +1,8 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="AlMamlakaTV.jo" status="online",Al Mamlaka TV (1080p)
 https://almamlka-live.ercdn.net/almamlka/almamlka.m3u8
+#EXTINF:-1 tvg-id="AlMamlakaTV.jo" status="online",Al Mamlaka TV (1080p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UC0jiFAzTgl17k7awGbuoYew/live
```

Comment:
> Nothing to do for https://www.youtube.com/channel/UC0jiFAzTgl17k7awGbuoYew/live, reason being No streams found.

Reply:
"Started streaming 115 minutes ago"

Reply:
This should be 24/7, I think the stream was restarted

Reply:
Not working again.

https://www.youtube.com/channel/UC0jiFAzTgl17k7awGbuoYew/live

<img width="492" alt="image" src="https://user-images.githubusercontent.com/7253922/187549661-98d45c08-b394-43a7-85d0-b15badc0263b.png">

Reply:
Thanks, deleted it